pkg/string/v1alpha1/string.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2018-2019 The OpenEBS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strings
```

Comment:
naming packages as standard ones causes issues that need to be imported with different name.. better to avoid these names
why we need this package?
making list is fine, but, costly operation

Reply:
So I thought of all list and string related operation can be done here.

Reply:
removed this package